apis/Google.Devtools.AspNet/Google.Devtools.AspNet/CloudTrace/BufferingTraceConsumer.cs

```diff
@@ -23,7 +23,7 @@ namespace Google.Devtools.AspNet
     /// </summary>
     internal sealed class BufferingTraceConsumer : IFlushableTraceConsumer
     {
-        // The default buffer size in bytes. 2^16 = 65536.
+        /// <summary>The default buffer size in bytes. 2^16 = 65536.</summary>
```

Comment:
I removed this.  I don't think it makes sense for a private field.  This also seems to match other files in the repo.